### PR TITLE
Is obsolete related issues

### PR DESCRIPTION
**Pull request by henrietteharmse**

### CODE REVIEW

**Comment by henrietteharmse** [comment]
I think it will be a good idea to do the testcase updates in a different commit in future.

File: frontend/src/components/Banner.tsx
Comment (haideriqbal) [inline frontend/src/components/Banner.tsx:1]: Are there any banner related changes regarding this PR?

File: frontend/dist/banner.txt
Comment (haideriqbal) [inline frontend/dist/banner.txt:1]: we should remove this whitespace/newline

**Review by haideriqbal — APPROVED** [review]
Have left some comments regarding banner changes... otherwise all good!

**Comment by henrietteharmse** [comment]
Yes, there were banner changes to fix #689.

**Pull request merged**